### ISSUE TITLE
Adding Multiple HDF files bug

### ISSUE THREAD

**Issue by ransom1122** [lubomyr/uae4arm#22]
Hi Lubo,

I have noticed when inserting a 2nd hdf into uae4arm hard drives tab it won't recognize the HDF unless you reset Uae4arm and then re attach 2nd hdf file. This also happens if attaching a 3rd or 4th HDF, you must reset uae4arm each time prior adding any HDF file after inserting the first HDF.

Can this be fixed please?

Regards
Ransom

**Comment by miker1117** [comment]
I can confirm this, although all I do 8s save the config and restart the app and the hdf/directory shows up....

Mike

**Comment by ransom1122** [comment]
Correct a bit annoying as to why a second attached hdf won't show unless uae4arm is rebooted

**Comment by miker1117** [comment]
Yes, and with the blinking red light (guru?) When switching configs or Roms, for testing purposes it'd probably be way more convenient if the 'reststart' button actually dropped the app from memory band restarted..
